weasyl/oauth2.py

```diff
@@ -82,7 +106,9 @@ def POST(self):
         return body
 
 
-def get_userid_from_authorization(scopes=['wholesite']):
+def get_userid_from_authorization(scopes=None):
+    if not scopes:
+        scopes = ['wholesite']
     valid, request = server.verify_request(*(extract_params() + (scopes,)))
```

Comment:
Not purely insanity, however I believe I may have been overestimating the dangers of mutable default arguments here. It's probably safe to just undo this..